### ISSUE TITLE
Add title attr to Google map iframes on single location template

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#196]
**Description**

See title

**Why it's Important**

`<iframe>`s should have a description of their content in a `title` attribute.

**Alternatives**

n/a

**Possible Implementation**

Add it

**Additional context**

n/a
